### PR TITLE
Data vis

### PR DESCRIPTION
**Pull request by sgiaccio**

### CODE REVIEW

File: common/i18n/resources/en.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ module.exports = {
     srs: 'SRS',
     errorMessage: 'Error message',
     errorMessage_plural: 'Error messages',
+    invalid: 'INVALID',
```
Comment (minotogna) [inline common/i18n/resources/en.js]: should the uppercase be managed via css?

File: common/surveyRdb/dataSort.js [hunk 2]
```diff
@@ -27,8 +27,8 @@ const getSortPreparedStatement = sortCriteria => {
 const findVariableByValue = (value) =>
   R.find(v => v.value === value)
 
-const getViewExpr = R.pipe(
-  R.map(c => `${c.label} ${c.order === keys.order.asc ? 'ASCENDING' : 'DESCENDING'}`),
+const getViewExpr = (ascLoc, descLoc) => R.pipe(
```
Comment (minotogna) [inline common/surveyRdb/dataSort.js]: what is loc? maybe label?

File: webapp/loggedin/surveyViews/surveyForm/navigation/formEntryActions.js [hunk 3]
```diff
@@ -27,32 +27,34 @@ const RecordEntryButtons = (props) => {
 
   const { i18n } = useContext(AppContext)
 
+  const getStepLabel = step => console.log(step) || i18n.t(`surveyForm.step.${RecordStep.getName(step)}`)
```
Comment (minotogna) [inline webapp/loggedin/surveyViews/surveyForm/navigation/formEntryActions.js]: let's not leave console.log in the master branch

File: common/surveyRdb/dataSort.js (hunk 2 above)
Comment (sgiaccio) [reply common/surveyRdb/dataSort.js]: Stands for localized, can be label too

File: webapp/loggedin/surveyViews/surveyForm/navigation/formEntryActions.js (hunk 3 above)
Comment (sgiaccio) [reply webapp/loggedin/surveyViews/surveyForm/navigation/formEntryActions.js]: right

File: common/i18n/resources/en.js (hunk 1 above)
Comment (sgiaccio) [reply common/i18n/resources/en.js]: Yes, changing to lowercase

File: common/surveyRdb/dataSort.js [hunk 4]
```diff
@@ -27,8 +27,8 @@ const getSortPreparedStatement = sortCriteria => {
 const findVariableByValue = (value) =>
   R.find(v => v.value === value)
 
-const getViewExpr = R.pipe(
-  R.map(c => `${c.label} ${c.order === keys.order.asc ? 'ASCENDING' : 'DESCENDING'}`),
+const getViewExpr = (ascLavel, descLabel) => R.pipe(
```
Comment (minotogna) [inline common/surveyRdb/dataSort.js]: it should be label, not lavel i guess

File: common/i18n/resources/en.js (hunk 1 above)
Comment (minotogna) [reply common/i18n/resources/en.js]: yes. let's also mange it via css

**Pull request merged**